### PR TITLE
jormun: Change log.warn() to log.warning() because warn() function is deprecated

### PR DESCRIPTION
**Pull request by benoit-bst**
Currently, there is a crash :
```
    "attribute refresh operation cannot proceed" % (state_str(state))
DetachedInstanceError: Instance <AutocompleteParameter at 0x7fc6635d34d0> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: http://sqlalche.me/e/bhk3)
tyr: [celery.worker.strategy] [2019-11-07 07:55:06,343] [INFO] [23953] Received task: tyr.binarisation.openaddresses2mimir[242c0586-44c2-486b-80d8-8b02f90d08bc]
tyr: [celery.worker.job] [2019-11-07 07:55:06,354] [ERROR] [23953] Task tyr.binarisation.openaddresses2mimir[242c0586-44c2-486b-80d8-8b02f90d08bc] raised unexpected: AttributeError("'LoggerAdapter' object has no
attribute 'warn'",)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 238, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/tyr/helper.py", line 86, in __call__
    return TaskBase.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 416, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/tyr/binarisation.py", line 637, in openaddresses2mimir
    logger.warn(
AttributeError: 'LoggerAdapter' object has no attribute 'warn'
```
it is normal because it depends of the python logging lib. there is one thing that is sure, **warn function is deprecated.**
```
logging.warning(msg, *args, **kwargs)
Logs a message with level WARNING on the root logger. The arguments are interpreted as for debug().
Note :
There is an obsolete function warn which is functionally identical to warning. As warn is deprecated, please do not use it - use warning instead.
```
This PR replace warn by warning function (My RipGrep helped me)

**Extra** :
I checked if it exists wrong log functions like `log.err` (True name is .error() ) or `log.crit` (True name is .critical()). I've found nothing

### CODE REVIEW

**Review by AurelienLP — APPROVED** [review]

**Review by kinnou02 — APPROVED** [review]

**Review by nonifier — APPROVED** [review]

**Comment by benoit-bst** [comment]
Jenkins, test this please !

File: source/jormungandr/jormungandr/parking_space_availability/abstract_provider_manager.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def _find_provider(self, poi):
     def _init_class(self, cls, arguments):
         try:
             if '.' not in cls:
-                self.log.warn('impossible to build, wrongly formated class: {}'.format(cls))
```
Comment (pbougue) [inline source/jormungandr/jormungandr/parking_space_availability/abstract_provider_manager.py:92]: You missed one just below, line 99 :wink: 
(don't know if it's worth a new pass with the regexp for other .err() or so)

**Review by pbougue — CHANGES_REQUESTED** [review]

File: source/jormungandr/jormungandr/parking_space_availability/abstract_provider_manager.py (hunk 1 above)
Comment (benoit-bst) [reply source/jormungandr/jormungandr/parking_space_availability/abstract_provider_manager.py:92]: Thanks :+1: , don't know where it was in my RipGrep response, :thinking:
probably a cache problem. 
It seems it was the only one

Comment (benoit-bst) [reply source/jormungandr/jormungandr/parking_space_availability/abstract_provider_manager.py:92]: recheck for err too

**Review by pbougue — APPROVED** [review]

**Pull request merged**